src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string name="app_name" translatable="false">Shadowsocks</string>
+  <string name="screen_name" translatable="false">shadowsocks ▾</string>
+
+  <!-- global/misc category -->
+  <string name="misc_cat">Разное</string>
+  <string name="profile">Профиль</string>
+  <string name="profile_summary">Переключить на другой профиль или добавить новые</string>
+  <string name="nat">Режим NAT (не рекомендуется)</string>
+  <string name="nat_summary">Использовать режим NAT вместо VPN. Требует ROOT прав.</string>
+  <string name="stat">Сетвая Активность</string>
+  <string name="stat_summary">Отправлено: \t\t\t\t\t%3$s\t↑\t%1$s/s\nПолучено: \t%4$s\t↓\t%2$s/s</string>
+  <string name="stat_profiles">\n%1$s↑\t%2$s↓</string>
+  <string name="traffic_summary">%1$s/s↑\t%2$s/s↓</string>
+  <plurals name="bytes">
+    <item quantity="one">Byte</item>
+    <item quantity="other">Bytes</item>
+  </plurals>
```

Comment:
Maybe we should use `one` (for 1, 21, 31, 41, 51, 61, 71, 81, 101, 1001, ...), `few` (for like 2, 3, 4) and `other` here?